src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -769,6 +771,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\MemoryMarshal.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\UnmanagedCallersOnlyAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\NativeLibrary.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\NFloat.cs" />
```

Comment:
I may have missed it, but I assume we want to avoid adding this to Mono? My concern is this https://github.com/dotnet/runtime/issues/13788#issuecomment-727920960.

Reply:
I think this is a question for @marek-safar (separately, is there a team for mono-contributions, like there is for jit-contrib?)

Reply:
This has to be added to Mono too. We do not want to have public surface differences between Mono and CoreCLR.

Dealing with binary compatibility for the legacy nint/nuint/nfloat Xamarin types is a separate problem: https://github.com/dotnet/runtime/issues/44736#issuecomment-728200317